eng/DotNetBuild.props

```diff
@@ -107,6 +107,9 @@
 
       <!-- Needed until https://github.com/dotnet/runtime/issues/109329 is fixed. -->
       <InnerBuildArgs Condition="'$(NetCoreAppToolCurrentVersion)' != ''">$(InnerBuildArgs) /p:NetCoreAppToolCurrentVersion=$(NetCoreAppToolCurrentVersion)</InnerBuildArgs>
+
+      <!-- Pass ShortStack to the inner build so we can skip building host-machine components for downstream repos when we're the last repo. -->
+      <InnerBuildArgs Condition="'$(ShortStack)' == 'true'">$(InnerBuildArgs) /p:ShortStack=true</InnerBuildArgs>
```

Comment:
We avoided passing the `ShortStack` property into the runtime repo build in favor of explicit control properties. Is that possible here?

Reply:
I see where you are coming from here. You don't want to build the host rid crossgen2 tooling if no other repo builds it after runtime and needs it.

Honestly, I wouldn't special case short stack builds here. We already build all libraries packages even in short stack builds but then don't use them anywhere. It's fine to overbuild in these cases as short stack builds already finish early and by not special casing, we keep the complexity low. We even talked about cases where aspnetcore would also build in short stack builds eventually.